src/api/eth_ens.rs

```diff
@@ -0,0 +1,827 @@
+//! `Eth` namespace, ens
```

Comment:
I think the file should be moved to `contract` sub-module, since it's a higher-level API to the ENS contract as far as I understand.

Reply:
Let me know what you think of the new organization.